### ISSUE TITLE
Doesn't work on IOS

### ISSUE THREAD

**Issue by mcourant** [mwitiderrick/text-recognition#1]
Hi, 

I can't use your project on iOS 13.6.

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.17.5, on Mac OS X 10.15.3 19D76, locale fr-FR)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 11.6)
[✓] Android Studio (version 4.0)
[✓] VS Code (version 1.47.3)
[✓] Connected device (1 available)

• No issues found!

Have you an idea ?

**Comment by mwitiderrick** [comment]
Hey I didn't test this code on IOS, I therefore can't guarantee that it
will or will not work there. What error do you have?

**Comment by mcourant** [comment]
I can't recognize a text in business card, that weird because that doesn't change between a pannel or a simple text.

**Comment by mwitiderrick** [comment]
Try it on an Android and IOS emulator

**Comment by mwitiderrick** [comment]
Did you get it to work?

**Comment by mcourant** [comment]
Yes, it's work but i've a problem when I have a logo or a text with a specific font. ML doesn't recognize and I don't know how to solve it :/
